base.ts

```
@@ -0,0 +1,307 @@
+      this.iframe = iframe;
+      iframe.id = LucraClientIframeId;
+      iframe.src = this.url;
+      iframe.style.height = "100%";
+      iframe.style.width = "100%";
+      iframe.allow =
+        "geolocation *; web-share; accelerometer *; bluetooth *; gyroscope *; clipboard-write *; payment;";
+      element.appendChild(iframe);
+    } catch (e) {
+      console.error("Error opening up LucraSports", e);
+    } finally {
+      return this;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)